en/releases/release-1.1.11.md

```diff
@@ -0,0 +1,25 @@
+---
+title: TiDB Operator 1.1.11 Release Notes
+---
+
+# TiDB Operator 1.1.11 Release Notes
+
+Release date: February 26, 2021
+
+TiDB Operator version: 1.1.11
+
+## New Features
+
+- Tune LeaderElection, support user configuration about LeaderElection ([#3794](https://github.com/pingcap/tidb-operator/pull/3794), [@july2993](https://github.com/july2993))
+- Add `tidb_cluster` label for the scrape jobs in TidbMonitor to support multiple clusters monitoring ([#3750](https://github.com/pingcap/tidb-operator/pull/3750), [@mikechengwei](https://github.com/mikechengwei))
+- Support setting customized store labels according to the node labels ([#3784](https://github.com/pingcap/tidb-operator/pull/3784), [@L3T](https://github.com/L3T))
+
+## Improvements
+
+- Add TiFlash rolling upgrade logic to avoid all TiFlash stores unavailable during upgrade ([#3789](https://github.com/pingcap/tidb-operator/pull/3789), [@handlerww](https://github.com/handlerww))
```

Comment:
```suggestion
- Add TiFlash rolling upgrade logic to avoid all TiFlash stores being unavailable at the same time during the upgrade ([#3789](https://github.com/pingcap/tidb-operator/pull/3789), [@handlerww](https://github.com/handlerww))
```